File: server/models/location.js
"use strict";

class Location {

    constructor (id, name, countrycode, wikiURL, infoURLs) {
        this.id = id;
        this.name = name;
        this.countrycode = countrycode;
        this.wikiURL = wikiURL;
        this.infoURLs = infoURLs;
    }

    static fromJSON (json) {
        let x = JSON.parse(json);
        return new Location(x.id, x.name, x.countrycode, x.wikiURL, x.infoURLs)
    }
}

File: src/main/java/com/eccff/launchbot/models/Location.java
package com.eccff.launchbot.models;

import lombok.Builder;
import lombok.Data;

import javax.persistence.Entity;
import javax.persistence.Id;
import java.util.List;

@Data
@Entity
@Builder
public class Location {

    @Id
    private int id;
    private String name;
    private String countrycode;
    private String wikiURL;
    private List<String> infoURLs;

}